src/components/PieChart/PieChart.tsx

```diff
@@ -1,41 +1,84 @@
-import { Animation, EventTracker } from '@devexpress/dx-react-chart';
-import {
-  Chart,
-  Legend,
-  PieSeries,
-  Title,
-  Tooltip,
-} from '@devexpress/dx-react-chart-material-ui';
-import { Paper } from '@material-ui/core';
-import React, { FC, useEffect, useState } from 'react';
-import PieChartDataProps from '../../models/PieChart/PieChartDataProps';
+import { Paper, useTheme } from '@material-ui/core';
+import React, { FC, useCallback, useMemo } from 'react';
+import { Pie } from 'react-chartjs-2';
 import PieChartProps from '../../models/PieChart/PieChartProps';
```

Comment:
PieChart não é uma entidade, e sim, apenas propriedade do React. O que for prop deve ser definido dentro do componente par evitar o acoplamento.

Reply:
Movi os arquivos de interface para a pasta do componente.